### PR TITLE
rename JwsVerificationKey2020 to JsonWebKey2020

### PR DESCRIPTION
**Pull request by kdenhartog**
Signed-off-by: Kyle Den Hartog <kyle.denhartog@mattr.global>

Closes issue w3c/vc-jws-2020#19

### CODE REVIEW

**Review by OR13 — CHANGES_REQUESTED** [review]
I'm in favor of these changes, but the context definition, and vocab defintion for the older term need to be retained with a deprecation warning.

**Comment by kdenhartog** [comment]
@OR13 fixed so it's deprecated rather than removed now. Tests are included as well for both and working for me.

File: docs/contexts/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ A public key in JWK format. A JSON Web Key (JWK) is a JavaScript Object Notation
 }
 ```
 
-<h4 id="JwsVerificationKey2020"><a href="#JwsVerificationKey2020">JwsVerificationKey2020</a></h4>
+<h4 id="JsonWebKey2020"><a href="#JsonWebKey2020">JsonWebKey2020</a></h4>
```
Comment (OR13) [inline docs/contexts/index.md:44]: This is where the term is defined... the deprecated term, has no defintion now. can you handle this the same way you did in did core, define the older term and warn about using it?

File: README.md [hunk 2]
```diff
@@ -112,6 +112,24 @@ See the [Linked Data Signature Suite Vocabulary](https://w3c-ccg.github.io/lds-j
 - [example keystore](https://w3c-ccg.github.io/lds-jws2020/example/didDocJwks.json).
 - [example did document](https://w3c-ccg.github.io/lds-jws2020/example/didDoc.json)
 
+#### Deprecated Examples
```
Comment (OR13) [inline README.md:115]: This is helpful

File: docs/contexts/index.md (hunk 1 above)
Comment (kdenhartog) [reply docs/contexts/index.md:44]: Sure, will get that fix in. Also is the table that registers these supposed to be generated by the code? If so, I'll update that as well so we actually register x25519 jwks.

Comment (OR13) [reply docs/contexts/index.md:44]: ahh, yes... that table does need to be updated. I suggest doing that in a separate PR

Comment (OR13) [reply docs/contexts/index.md:44]: To be super clear... I will comntinue to request changes, until I see the old term defintion preserved.

**Review by kimdhamilton — APPROVED** [review]
+1 to this change, with the caveat of addressing @OR13's concerns about deprecating the old term

File: docs/contexts/index.md (hunk 1 above)
Comment (kdenhartog) [reply docs/contexts/index.md:44]: Yup makes sense. Working on fixing them up today now that I've gotten a bit of time to get back to this.

**Comment by kdenhartog** [comment]
@OR13 ok I fixed the definition problem from the issue above. Can you review again to make sure every place is updated properly?

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
@kdenhartog thank you!

**Pull request merged**